run_pipeline.sh

```diff
@@ -1,72 +1,79 @@
 #!/bin/bash
-# Note, change the directory to where you want these things to appear before running this code.
+# C. Savonen
+# CCDL for ALSF 2018
 
-# Make directories
-mkdir data
-mkdir data/raw_data
-mkdir data/aligned_reads
-mkdir data/fastqc_reports
-mkdir data/fastqc_trimmed
-mkdir results
-mkdir results/map_qc
-
-#--------------------------- Download fastq data-------------------------------#
-# Note: because running all ~3800 samples takes quite a bit of time, this 
-# example run is set to only run 10 samples. When you want to run the full set, 
-# delete the -n argument.  
-Rscript 0-download_fastq_data.R \
--i SRP079058 \
--d data/raw_data \
--n 20 
-
-#------------------------Run fastqc and get reports----------------------------#
-# Run sequence quality control with FASTQC
-/FastQC/fastqc data/raw_data/* --outdir data/fastqc_reports
+# Purpose: running the pre-processing steps for single cell data. 
 
-# Obtain summary report:
-Rscript 1-get_fastqc_reports.R -d data/fastqc_reports -o results
+#----------------------------Make directories---------------------------------#
+mkdir darmanis_data
```

Comment:
Does it make sense to set all of these directories as variables at the top of the script? So, for example, you could use `${salmon_directory}` throughout